charts/eks-anywhere-packages-migrations/templates/adopt-crds-awssecret.yaml

```diff
@@ -58,6 +58,10 @@ spec:
     spec:
       restartPolicy: Never
       serviceAccountName: {{ .Values.serviceAccount.name }}
```

Comment:
This isn't in the values.yaml how would someone know to add this?

Reply:
(note that this might not even be required if we release the oci auth package at the same time)